phpBB/styles/prosilver/template/mcp_topic.html

```diff
@@ -120,7 +120,7 @@ <h3><a href="{postrow.U_POST_DETAILS}">{postrow.POST_SUBJECT}</a></h3>
 				<p class="author">
 					<a href="#pr{postrow.POST_ID}" title="{postrow.MINI_POST}">
 						<i class="icon fa-file fa-fw icon-lightgray icon-tiny" aria-hidden="true"></i><span class="sr-only">{postrow.MINI_POST}</span>
-					</a> {L_POSTED} {postrow.POST_DATE} {L_POST_BY_AUTHOR} <!-- EVENT mcp_topic_postrow_post_details_author_username_prepend --><strong>{postrow.POST_AUTHOR_FULL}</strong><!-- EVENT mcp_topic_postrow_post_details_author_username_append --><!-- IF postrow.U_MCP_DETAILS --> [ <a href="{postrow.U_MCP_DETAILS}">{L_POST_DETAILS}</a> ]<!-- ENDIF -->
+					</a> {L_POSTED} {postrow.POST_DATE} {L_POST_BY_AUTHOR} {% EVENT mcp_topic_postrow_post_details_author_username_prepend %}<strong>{postrow.POST_AUTHOR_FULL}</strong>{% EVENT mcp_topic_postrow_post_details_author_username_append %}<!-- IF postrow.U_MCP_DETAILS --> [ <a href="{postrow.U_MCP_DETAILS}">{L_POST_DETAILS}</a> ]<!-- ENDIF -->
```

Comment:
´mcp_topic_post_author_full_prepend´ ?

Reply:
What about `mcp_topic_postrow_author_full_prepend` ???

Reply:
Whý? That's a part of the redundancies, there can be only one _postrow_ in there. ;)

Reply:
Yea!! I just replaced your `_post_` with `_postrow_` ... to make it more close to `mcp_topic_postrow_post_details_before` ... & it has only one `postrow`...